octproz/src/glwindow2d.cpp

```diff
@@ -253,24 +253,25 @@ void GLWindow2D::initializeGL(){
 }
 
 void GLWindow2D::paintGL(){
-	glLoadIdentity();
-	glTranslatef(this->xTranslation, this->yTranslation, 0);
-	glRotatef(this->rotationAngle, 0.0, 0.0, 1.0);
-	glScalef(this->scaleFactor, this->scaleFactor, 0.f);
-	glBindBuffer(GL_PIXEL_UNPACK_BUFFER_ARB, this->buf);
-	glBindTexture(GL_TEXTURE_2D, this->texture);
-	glTexSubImage2D(GL_TEXTURE_2D, 0, 0, 0, this->width, this->height, GL_DEPTH_COMPONENT, GL_FLOAT, 0);
-	glBindBuffer(GL_PIXEL_UNPACK_BUFFER_ARB, 0);
-	glEnable(GL_TEXTURE_2D);
-	glColor3f(1.0, 1.0, 1.0);
-	glBegin(GL_QUADS);
-		glTexCoord2f(0, 1), glVertex2f(-this->screenWidthScaled, -this->screenHeightScaled); //place upper left texture coordinate to bottom left screen position
-		glTexCoord2f(1, 1), glVertex2f(-this->screenWidthScaled, this->screenHeightScaled); //upper right texture coordinate to upper left screen position
-		glTexCoord2f(1, 0), glVertex2f(this->screenWidthScaled, this->screenHeightScaled); //bottom right texture coordinate to upper right screen position
-		glTexCoord2f(0, 0), glVertex2f(this->screenWidthScaled, -this->screenHeightScaled); //bottom left texture coordinate to bottom right screen position
-	glEnd();
-	glDisable(GL_TEXTURE_2D);
-	glBindTexture(GL_TEXTURE_2D, 0);
+    glLoadIdentity();
+    glTranslatef(this->xTranslation, this->yTranslation, 0); //verschieben
+//    glRotatef(this->rotationAngle, 0.0, 0.0, 1.0);
+    glScalef(this->scaleFactor, this->scaleFactor, 0.f); //zoom
+    glBindBuffer(GL_PIXEL_UNPACK_BUFFER_ARB, this->buf);
+    glBindTexture(GL_TEXTURE_2D, this->texture);
+    glTexSubImage2D(GL_TEXTURE_2D, 0, 0, 0, this->width, this->height, GL_DEPTH_COMPONENT, GL_FLOAT, 0);
+    glBindBuffer(GL_PIXEL_UNPACK_BUFFER_ARB, 0);
+    glEnable(GL_TEXTURE_2D);
+    glColor3f(1.0, 1.0, 1.0); //Backgroundcolor
+    glBegin(GL_QUADS);
+        glTexCoord2f(0, 1), glVertex2f(this->screenWidthScaled01,this->screenHeightScaled01); //place upper left texture coordinate to bottom left screen position
+        glTexCoord2f(1, 1), glVertex2f(this->screenWidthScaled11, this->screenHeightScaled11); //upper right texture coordinate to upper left screen position
+        glTexCoord2f(1, 0), glVertex2f(this->screenWidthScaled10,this->screenHeightScaled10); //bottom right texture coordinate to upper right screen position
+        glTexCoord2f(0, 0), glVertex2f(this->screenWidthScaled00, this->screenHeightScaled00); //bottom left texture coordinate to bottom right screen position
+    glEnd();
+    glDisable(GL_TEXTURE_2D);
+    glBindTexture(GL_TEXTURE_2D, 0);
+

```

Comment:
humble request: could you use tabs instead of spaces, please? Just to keep the code more or less consistent.
suggestion: please write comments in English if you think they are necessary.